subprojects/android/android.gradle

```diff
@@ -0,0 +1,8 @@
+ext {
+    artifactName = 'mockito-android'
+    bintrayAutoPublish = false
+    mavenCentralSync = false
+}
+
+apply from: "$rootDir/gradle/java-library.gradle"
+apply from: "$rootDir/gradle/publishable-java-library.gradle"
```

Comment:
The aspiration of this PR is to make new module publishable, not to implement android submodule :) When we implement the module, it probably only needs single dependency on root project, e.g:

```groovy
dependencies {
  compile project(":")
}
```